.yarn/versions/2930fda4.yml

```diff
@@ -0,0 +1,35 @@
+releases:
+  "@yarnpkg/cli": patch
+  "@yarnpkg/fslib": patch
+  "@yarnpkg/pnp": patch
```

Comment:
Shouldn't `plugin-pnp` get a patch as well since its the one writing the pnp hook?

Reply:
No, it depends on `@yarnpkg/pnp` via a caret hook, so it'll pick up the fix without release.

Reply:
For a fresh install sure, but `yarn up @yarnpkg/*` wouldn't update `@yarnpkg/plugin-pnp -> @yarnpkg/pnp`

Reply:
True, but so far packages haven't been updated because of their transitive dependencies, unless required to like for the CLI bundle (which can't upgrade transitive deps) or the hook itself (same thing).